app/assets/javascripts/comfortable_mexican_sofa/admin/modules/CategoryExpand.js

```diff
@@ -0,0 +1,76 @@
+define(['jquery','DoughBaseComponent'], function ($, DoughBaseComponent) {
+'use strict';
+
+  var defaultConfig = {
+    categoriesCharacterCount: 20
+  };
+  var CategoryExpandProto;
+
+  function CategoryExpand($el, config) {
+    CategoryExpand.baseConstructor.call(this, $el, config, defaultConfig);
+  }
+
+  DoughBaseComponent.extend(CategoryExpand);
+
+  CategoryExpand.componentName = 'CategoryExpand';
+
+  CategoryExpandProto = CategoryExpand.prototype;
```

Comment:
If this was done to reduce typing/ readability, good idea... An alternative to this is shown below although I've not seen it in this code-base yet...

```
CategoryExpand.prototype = {
   'init' = function() {
    },
   'aMethod' = function() {
    },
   'anotherMethod' = function() {
    }
}
```

This is just an FYI...